src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -289,10 +299,10 @@
     </Compile>
     <Compile Include="$(CommonPath)\System\Security\Cryptography\DerSequenceReader.cs">
       <Link>Common\System\Security\Cryptography\DerSequenceReader.cs</Link>
-    </Compile>
+    </Compile>    
   </ItemGroup>
   <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
+      <None Include="project.json" />
```

Comment:
I know this isn't related to your change but this include of project.json likely shows a missing file in VS as it doesn't exist as all the project.json files are in subfolders.